app/assets/stylesheets/container_topology.css

```diff
@@ -124,3 +124,36 @@ g.EntityLegend {
     fill: black;
 }
 
+.canvas {
```

Comment:
Can you add the `.container_topology ` prefix for `.canvas`, `.popup` and the alert please? (See #6081.)